public/js/helpers/zoom_helper.js

```diff
@@ -112,19 +126,18 @@ export default class Zoom {
 
   // mapKey returns the corresponding map key, if the zoom meets the correct
   // range and position within the limits, else null.
-  static mapKey (zoom, limits, scale) {
+  static mapKey (zoom, limits, scale, isTimeAxis, ratio) {
```

Comment:
This doesn't need to be changed. The scale parameter already handled the transformation, it just needs to be set correctly.  Use `scale = 300000` (300 seconds per block) when it is the blocks axis.